### PR TITLE
Return All Payees with Balances

### PR DESCRIPTION
**Pull request by ayshptk**

### CODE REVIEW

**Comment by atbe** [comment]
@ayshptk please add:

1. Proper docstrings
2. Proper tests

File: src/modules/royalty.ts [hunk 1]
```diff
@@ -134,6 +134,26 @@ export class SplitsModule extends Module<Royalty> implements ISplitsModule {
     return recipients;
   }
 
+  public async getAllRecipientsWithBalance() {
```
Comment (atbe) [inline src/modules/royalty.ts:137]: Rename to `balanceOfAllRecipients`

File: src/modules/royalty.ts [hunk 2]
```diff
@@ -134,6 +134,26 @@ export class SplitsModule extends Module<Royalty> implements ISplitsModule {
...
+    const recipients = await this.getAllRecipients();
+    const balances: { [key: string]: BigNumber } = {};
+    recipients.forEach(async (recipient) => {
+      balances[recipient.address] = await this.balanceOf(recipient.address);
+    });
+    return balances;
+  }
+
+  public async getAllRecipientsWithTokenBalance(tokenAddress: string) {
```
Comment (atbe) [inline src/modules/royalty.ts:146]: Rename to `balanceOfTokenAllRecipients`

**Comment by ayshptk** [comment]
yeah adding docstrings, will add tests after fixing bug

**Review by jakeloo — APPROVED** [review]

**Pull request merged**